### DISCUSSION TITLE
Social login safari

### DISCUSSION THREAD

**Discussion by cinthya-trejo-occ-com-mx** [ory/kratos#3874] [Q&A]
I am using google as a social sign-in provider. I would like to know if this behavior is correct following these steps in incognito Safari browser.

1. User clicks google button.
2. Google shows the account login interface.
3. User fills email and password.
4. User logs out.
5. When the user clicks again on the google button to log in, the google form is not displayed and the user logs in automatically.

Thanks!

**Comment by vinckr** [comment]
Hello @Yasbek 

>User logs out.

Can you clarify that step?

Does the user log out of Google or out of your application that uses login with Google? 

If they log out of your application and stay logged in at Google this is the expected behaviour. 

Essentially Google knows that this users has given consent to your application to use their Google account to login to your application.
